client/src/components/Modal/__tests__/MapModal.js

```diff
@@ -0,0 +1,7 @@
+// canot run tests on MapModal due to react-native-maps not liking jest
```

Comment:
actually its because it doesn't come with built in mocks but we should look into creating our own simple version.